spec/support/rails_template.rb

```diff
@@ -114,6 +114,8 @@ class Tagging < ActiveRecord::Base
 
 RUBY
 
+inject_into_file 'config/environments/test.rb', "\n  config.autoloader = :classic\n", after: 'Rails.application.configure do' if Rails::VERSION::MAJOR >= 6
```

Comment:
Can we add a TODO note about fixing this in the future, maybe with a reference to the STI cucumber feature that fails with `zeitwerk`?

Reply:
I added the TODO note